.github/workflows/R-test-coverage.yml

```diff
@@ -0,0 +1,50 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
```

Comment:
NIT: Is it necessary to specify `master`? I think most/ all of our repositories consistently use `main` at this stage (maybe a good indication, if they don't, to switch them)

Reply:
this is leftover from the default R-Lib template. When we switch this repo to using RMI-PACTA/actions, this whole file will be removed, so I'm not worried at this point.